platform/broadcom/sonic-platform-modules-inspur/sc5630el/modules/inspur-pca9641.c

```diff
@@ -0,0 +1,361 @@
+/*
+ * I2C master demultiplexer driver for PCA9641 
```

Comment:
why need custom kernel driver, there is vanilla pca9641.c driver in the kernel.

Reply:
Hi @lguohan ,

We have our own proprietary hardware design, so we have to provide our driver.

Reply:
OK we will use Linux native driver.

Reply:
@lguohan  there is NO vanilla pca9641.c but **pca9541.c**, do u mean we need to change to this one?